redi-connect-front/src/components/atoms/Icon.scss

```diff
@@ -11,7 +11,15 @@
     width: 1.5rem;
     height: 1.5rem;
   }
-  
+
+  &--large {
+    height: 4rem;
+  }
+
+  &--default {
```

Comment:
Does not say a lot about the behaviour it stays for. We also it doesn''t really fit in to sizes... Please check if we could solve this without a generic size like this, since default is not really a size.

Reply:
I was thinking of 3 options: 
1. Another component e.g. `Image` which basically does the same thing than `Icon` but for the "bigger" images like in the hero, landing, prefooter split-menu etc. (also the name would fit better because it's not really an icon anymore)
2. Name it something like `&--original`
3. give the `icon` class `height: 100%` and `width: auto` and remove the default `const iconSize = size || 'medium'`